dist/index.d.ts

```diff
@@ -2968,7 +2983,7 @@ interface PowerCreep extends RoomObject {
     /**
      * Apply one of the creep's powers on the specified target.
      */
-    usePower(power: PowerConstant, target: RoomObject | RoomPosition): ScreepsReturnCode;
+    usePower(power: PowerConstant, target?: RoomObject | RoomPosition): ScreepsReturnCode;
```

Comment:
usePower target cannot be RoomPosition.

Reply:
The documentation says it accepts a RoomPosition. I brought this up on slack. We'll see if this was a typo.

Reply:
I agree after reviewing the current implementation of the intent. If RoomPositions are actually supported for future powers, this can be added back later.